frontend/src/scenes/users/Person.js

```diff
@@ -116,7 +121,30 @@ function _Person({ _: distinctId, id }) {
                 <br />
                 <br />
             </div>
-            <Events fixedFilters={{ person_id: person.id }} />
+            <Tabs
+                defaultActiveKey={activeTab}
+                onChange={(tab) => {
+                    setActiveTab(tab)
+                }}
+            >
+                <TabPane
+                    tab={<span data-attr="people-types-tab">Events</span>}
+                    key="events"
+                    data-attr="people-types-tab"
+                ></TabPane>
+                {window.posthog && window.posthog.isFeatureEnabled('session-recording-player') && (
+                    <TabPane
+                        tab={<span data-attr="people-types-tab">Sessions By Day</span>}
+                        key="sessions"
+                        data-attr="people-types-tab"
+                    ></TabPane>
```

Comment:
Nit: Should this line be `/>` (self-closing tab). Applies for the above TabPane as well.